index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="assets/utiles.css">
+    <title>Event Website</title>
+</head>
+
+<body>
+    <header class="header">
+
+        <!-- Desktop menu -->
+        <div class="header-top">
+            <ul>
+                <li><a href=""><img src="assets/icones/1.png" alt="github" width="30" height="30"></a></li>
+                <li><a href=""><img src="assets/icones/4.png" alt="tweeter" width="30" height="30"></a></li>
+                <li><a href="">Login</a></li>
+                <li><a href="">Sign Up</a></li>
+
+            </ul>
+
+        </div>
+        <div class="desktop-menu">
+            <a href="index.html"><img src="assets/images/GE.png" alt="" width="120" class="logo"></a>
+            <nav>
+                <ul class="desktop-nav">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Support</a></li>
+                    <li><a href="">News</a></li>
+                    <li class="ge-campaigne"><a href="">GE campaign</a></li>
+
+                </ul>
+            </nav>
+
+        </div>
+        <button class="menu-button">
+            <img src="assets/icones/menu.png" alt="menu" width="45" height="45">
+        </button>
+
+        <div class="nav-active">
+            <button class="menu-button">
+                <img src="assets/icones/cancel.png" alt="close menu" width="40" height="40">
+            </button>
+            <ul class="list-nav">
+                <li><a href="index.html" class="nav-list">Home</a></li>
+                <li><a href="about.html" class="nav-list">About</a></li>
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Support</a></li>
+                <li><a href="">News</a></li>
+            </ul>
+
+        </div>
+    </header>
+    <section class="head-line-section">
+        <div class="head-line-container">
+            <h2 class="orange">"Hello sharing world"</h2>
+            <h1 class="big-title">GOLDEN EVENT WEBSITE </h1>
+
+            <p class=" headline-p">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Aliquam atque, similique
+                nobis quasi facilis dignissimos inventore delectus, eaque pariatur modi est laudantium doloribus laborum
+                quis ipsa non sint facere explicabo?</p>
```

Comment:
### Your Paragraph
![box-background](https://user-images.githubusercontent.com/93039359/211143083-07ef0396-18ec-48c1-ba7f-71cdd27b586f.PNG)

### Design Paragraph
![box-background](https://user-images.githubusercontent.com/93039359/211143090-f4203ff2-4131-44c6-9863-3b87463eca5a.PNG)

- [ ] Your paragraph box should have a background color and white border which you can see in the design 🤔 Kindly follow the Cindy's design 😊